docs/luigi-core-api.md

```diff
@@ -87,14 +87,15 @@ Tells Luigi that the configuration has been changed. Luigi will update the appli
     <p>
     <ul>
       <li><code>navigation</code>: the navigation part of the configuration was changed. This includes navigation nodes, the context switcher, the product switcher and the profile menu.</li>
-      <li><code>navigation.nodes</code>: navigation nodes were changed.</li>
       <li><code>navigation.contextSwitcher</code>: context switcher related data were changed.</li>
       <li><code>navigation.productSwitcher</code>: product switcher related data were changed.</li>
       <li><code>navigation.profile</code>: profile menu was changed.</li>
       <li><code>settings</code>: settings were changed.</li>
       <li><code>settings.header</code>: header settings (title, icon) were changed.</li>
       <li><code>settings.footer</code>: left navigation footer settings were changed.</li>
     </ul>
+    <!-- add-attribute:class:warning -->
+    >**NOTE:** <code>navigation.nodes</code> is temporary deprecated due to unwanted side effects. Therefore do not use `getConfig` and `setConfig` methods for updating `navigation.nodes` configuration.
```

Comment:
```suggestion
>**NOTE:** <code>navigation.nodes</code> is temporary deprecated due to unwanted side effects. Therefore do not use `getConfig` and `setConfig` methods for updating `navigation.nodes` configuration.
```
This note is not rendered correctly on the website, possibly because it is below the list. Maybe deleting the indent can fix it?